pkg/tenant/api/v1/api.go

```diff
@@ -174,6 +174,7 @@ type Organization struct {
 }
 
 type Tenant struct {
+	OrgID           string `json:"org_id"`
```

Comment:
Before updating all of the handlers, I thought it would be best to discuss if we want to add the `OrgID` to each resource on the client side.

Reply:
The OrgID on the client-side will principally come from the claims; so I think the user doesn't need to additionally specify the orgID - just the ID of the resource should be sufficient.